### PR TITLE
20201022 test step file

### PR DESCRIPTION
**Pull request by changlong-liu**
for issue #595

### CODE REVIEW

File: src/test/scenarios/synapse/output/coredefault/src/azure-cli/requirements.py3.Darwin.txt [hunk 1]
```diff
@@ -88,7 +88,7 @@ cffi==1.14.0
 chardet==3.0.4
 colorama==0.4.1
 ConfigArgParse==0.14.0
-cryptography==3.2
+cryptography==2.8
```
Comment (qiaozha) [inline src/test/scenarios/synapse/output/coredefault/src/azure-cli/requirements.py3.Darwin.txt:91]: I believe 3.2 is correct version. Could you merge the latest master?

File: src/test/scenarios/synapse/output/coredefault/src/azure-cli/requirements.py3.Linux.txt [hunk 2]
```diff
@@ -88,7 +88,7 @@ cffi==1.14.0
 chardet==3.0.4
 colorama==0.4.1
 ConfigArgParse==0.14.0
-cryptography==3.2
+cryptography==2.8
```
Comment (qiaozha) [inline src/test/scenarios/synapse/output/coredefault/src/azure-cli/requirements.py3.Linux.txt:91]: same here

File: src/test/scenarios/synapse/output/coredefault/src/azure-cli/requirements.py3.Darwin.txt (hunk 1 above)
Comment (changlong-liu) [reply src/test/scenarios/synapse/output/coredefault/src/azure-cli/requirements.py3.Darwin.txt:91]: yes, will merge and regenerate test

Comment (changlong-liu) [reply src/test/scenarios/synapse/output/coredefault/src/azure-cli/requirements.py3.Darwin.txt]: Hi @qiaozha , updated

**Review by qiaozha — APPROVED** [review]

**Pull request merged**